src/frameName.cpp

```diff
@@ -44,6 +44,8 @@ Matcher::Matcher(const Matcher& m) {
 }
 
 Matcher& Matcher::operator=(const Matcher& m) {
+    if(this == &m) return *this;
```

Comment:
Why didn't linter complain about no space after `if`? :)

Reply:
`clang-format` would do that, but unlike `clang-tidy` there's no option to just check for one thing. It's designed to execute all checks

Reply:
259c11ef2f3626e78e89563b516a6854bf8d01c5